.gitignore

```diff
@@ -2,3 +2,4 @@
 .dart_tool
 .packages
 pubspec.lock
+.idea
```

Comment:
Please remove this line! It should be in your `~/.gitignore_global`

Reply:
Removed!